motionplan/planManager.go

```diff
@@ -150,6 +150,7 @@ func (pm *planManager) PlanSingleWaypoint(ctx context.Context,
 	if err != nil {
 		return nil, err
 	}
+	pm.planOpts = opt
```

Comment:
This was an oversight before. The planManager is a planner just like the sub-planners it spawns, but it never actually had its options properly initialized. Which wasn't an issue back when everything had all the same defaults, but since options are initialized slightly differently for relative solving, this needs to now exist (or else use the sub-planner's options)